adapter/stackdriver/helper/common.go

```diff
@@ -35,3 +37,12 @@ func ToOpts(cfg *config.Params) (opts []gapiopts.ClientOption) {
 	}
 	return
 }
+
+// ToStringMap converts a map[string]interface{} to a map[string]string using fmt.Sprintf(%v)
+func ToStringMap(in map[string]interface{}) map[string]string {
```

Comment:
curious: is this also in a current outstanding PR too?  I feel like i've seen this before, but I can't tell if i'm just going crazy.

Reply:
Yeah - this PR is a superset of https://github.com/istio/mixer/pull/1235, which is where this change is actually from.